testsuite/features/reposync/reference_srv_check_reposync.feature

```diff
@@ -6,8 +6,8 @@ Feature: Reposync works as expected
 @scc_credentials
 @susemanager
   Scenario: Check reposync of Rocky Linux 8 channels being finished
-    Then I wait until the channel "rockylinux-8-x86_64" has been synced
-    And I wait until the channel "rockylinux-8-appstream-x86_64" has been synced
+    Then I wait until the channel "rockylinux8-x86_64" has been synced
```

Comment:
wow. i did not expect that one.

Reply:
No possible channel exist with `-` in rockylinux name (checked on uyuni:master)
```
wiktor-srv:~ # spacewalk-common-channels --list | grep rockylinux
 rockylinux8:         x86_64, aarch64
 rockylinux8-appstream: x86_64, aarch64
 rockylinux8-epel8:   x86_64, ppc64le, aarch64
 rockylinux8-extras:  x86_64, aarch64
 rockylinux8-ha:      x86_64, aarch64
 rockylinux8-powertools: x86_64, aarch64
 rockylinux8-uyuni-client: i386, x86_64, aarch64
 rockylinux8-uyuni-client-devel: i386, x86_64, aarch64
 rockylinux9:         x86_64, aarch64, ppc64le, s390x
 rockylinux9-altarch-common: aarch64
 rockylinux9-altarch-rockyrpi: aarch64
 rockylinux9-appstream: x86_64, aarch64, ppc64le, s390x
 rockylinux9-crb:     x86_64, aarch64, ppc64le, s390x
 rockylinux9-epel9:   x86_64, aarch64, ppc64le, s390x
 rockylinux9-extras:  x86_64, aarch64, ppc64le, s390x
 rockylinux9-highavailability: x86_64, aarch64, ppc64le, s390x
 rockylinux9-nfv:     x86_64, aarch64, ppc64le, s390x
 rockylinux9-plus:    x86_64, aarch64, ppc64le, s390x
 rockylinux9-resilientstorage: x86_64, aarch64, ppc64le, s390x
 rockylinux9-rt:      x86_64, aarch64, ppc64le, s390x
 rockylinux9-sap:     x86_64, aarch64, ppc64le, s390x
 rockylinux9-saphana: x86_64, aarch64, ppc64le, s390x
 rockylinux9-uyuni-client: i386, x86_64, aarch64, ppc64le, s390x
 rockylinux9-uyuni-client-devel: i386, x86_64, aarch64, ppc64le, s390x
```

Reply:
After running the scenario:
```
  Scenario: Add Rocky Linux 8                                                                # features/build_validation/reposync/srv_sync_all_products.feature:599
      This scenario ran at: 2024-02-29 12:01:56 +0000
    When I use spacewalk-common-channel to add all "rockylinux8" channels with arch "x86_64" # features/step_definitions/command_steps.rb:182
      Channel rockylinux8 added
      Channel rockylinux8-appstream added
      Channel rockylinux8-extras added
      Channel rockylinux8-uyuni-client-devel added

...
```

Notable fragments of `wiktor-srv:/var/log/rhn/reposync/rockylinux8-x86_64.log`:

```
2024/02/29 13:02:05 +02:00   Importing comps file 71f62d6dadfbf3238ce701da43cb69958ce4c546cc370f92e70ba933f3193c23-comps-BaseOS.x86_64.xml.
2024/02/29 13:02:05 +02:00   Renaming non-standard filename 71f62d6dadfbf3238ce701da43cb69958ce4c546cc370f92e70ba933f3193c23-comps-BaseOS.x86_64.xml to comps.x86_64.xml.
2024/02/29 13:02:05 +02:00 *** NOTE: Importing comps file for the channel 'rockylinux8-x86_64'. Previous comps will be discarded.
2024/02/29 13:02:05 +02:00 Repo URL: https://mirrors.rockylinux.org/mirrorlist?repo=BaseOS-8&arch=x86_64
2024/02/29 13:02:05 +02:00     Packages in repo:              2282
2024/02/29 13:02:08 +02:00     Packages already synced:          0
2024/02/29 13:02:08 +02:00     Packages to sync:              2282
2024/02/29 13:02:09 +02:00     New packages to download:      2282
2024/02/29 13:02:09 +02:00   Downloading packages:
2024/02/29 13:02:10 +02:00     1/2282 : ModemManager-1.20.2-1.el8.x86_64.rpm
...
2024/02/29 13:08:55 +02:00 Sync completed.
```
Full file for reference: 
[rockylinux8-x86_64.log](https://github.com/uyuni-project/uyuni/files/14449124/rockylinux8-x86_64.log)

I guess this one is ok.

Reply:
ok then no issue. I don't know how I (or someone else) invented that dash.